FileLiberator/AaxcDownloadDecrypt/DownloadBookDummy.cs

```diff
@@ -0,0 +1,19 @@
+﻿using DataLayer;
+using Dinah.Core.ErrorHandling;
+using System.Threading.Tasks;
+
+namespace FileLiberator.AaxcDownloadDecrypt
+{
+    public class DownloadBookDummy : DownloadableBase
+    {
+        public override async Task<StatusHandler> ProcessItemAsync(LibraryBook libraryBook)
+        {
+            return new StatusHandler();
```

Comment:
minutia: Task.FromResult creates a "pre-computed" task which avoids incurring some the async state machine's overhead:

public override /* also 'async' no longer necessary */ Task<StatusHandler> ProcessItemAsync(LibraryBook libraryBook) => Task.FromResult(new StatusHandler());